### PR TITLE
Document readability and spelling updates

### PR DESCRIPTION
**Pull request by MohibAyub**

### CODE REVIEW

**Comment by ColCarroll** [comment]
Thanks for spotting the `matplotlib` spelling! It looks like this is also adding whitespace to the start of lines for part of the rest of the readme. Can you remove that before we merge?

File: README.md [hunk 1]
```diff
@@ -144,4 +144,4 @@ There is also a Dockerfile which helps for isolating build problems and local de
 2. Clone this repo,
 3. Run `./scripts/start_container.sh`
 
- This should start a local docker container called arviz, as well as a Jupyter notebook server running on port 8888. The notebook should be opened in your browser automatically (you can disable this by passing --no-browser). The container will be running the code from your local copy of arviz, so you can test your changes.
+This should start a local docker container called arviz, as well as a Jupyter notebook server running on port 8888. The notebook should be opened in your browser automatically (you can disable this by passing --no-browser). The container will be running the code from your local copy of arviz, so you can test your changes.
```
Comment (canyon289) [inline README.md]: If we're being really nitpicky

> local copy of arviz, so 

This should be ArviZ since it's referring to the library

Comment (canyon289) [reply README.md]: You actually don't need to change this one anymore. This section got rewritten in another pr :)

**Comment by canyon289** [comment]
@MohibAyub Still working on this one?

**Comment by MohibAyub** [comment]
@canyon289 Not anymore. You can close the pull request

**Pull request closed without merging**